app/src/main/java/com/github/factotum_sdp/factotum/ui/directory/ContactDetailsFragment.kt

```diff
@@ -25,10 +33,26 @@ class ContactDetailsFragment : Fragment() {
             view.findNavController().navigate(R.id.action_contactDetailsFragment2_to_directoryFragment)} // go back to the list of contacts when the button is clicked
         setContactDetails(mainView, ContactsList.getItems()[arguments?.getInt("id")!!]) //links the contact details to the layout
 
-
         return mainView
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<Button>(R.id.run_button).setOnClickListener {
+            val route = DUMMY_ROUTE[0] //TODO : remove when merged with contact creation and use real route
```

Comment:
I completely agree this is a TODO, however since we don't want any TODO comments in our main branch (supposed to be the final product), I would either duplicate the branch and merge this branch without the TODOs or wait for the needed code to come available before merging this branch.